### DISCUSSION TITLE
I had an ERROR even though I have a license key, what should I do?

### DISCUSSION THREAD

**Discussion by suissa** [open-wa/wa-automate-nodejs#3119] [General]
I just purchased a license and I'm experiencing the following error: 
 🔶 ERROR: This feature requires an Insiders license. Use this link to get the correct license => https://get.openwa.dev/l/5511994649923?v=i&f=sendListMessage 🔶 

Trying: node wallet.test.js -l <LICENSE_KEY>
and  node wallet.test.js -license-key <LICENSE_KEY>

**Comment by suissa** [comment]
I tried in create too
create({
  licenseKey: "LICENSE_KEY"
})

**Reply by luizbizzio** [reply]
What feature did you try to use? It happens to all insiders features?

 I am not sure, but maybe some features you need another license key
